PaloAltoNetworks Device Security (SCM) Content Pack (#44866)
demisto/content: Packs/PaloAltoNetworks_DeviceSecurity/Integrations/PaloAltoNetworks_DeviceSecurity/PaloAltoNetworks_DeviceSecurity_description.md

> ## Get your Palo Alto Networks Device Security Credentials

Author: yes, updated with the title of the integration name and made the credential text to subheader

This integration requires API access configuration.

> To obtain the **Client ID** and **Client Secret**, see the [API User Guide](https://pan.dev/iot/api/iot-public-api-new).

Author: I have updated the detail steps for each of the below section. so, users can just follow along and set it up without leaving the platform.

Sections that i have covered are:

## Palo Alto Networks Device Security

### Required Permissions

### Get your Palo Alto Networks Device Security Credentials

### Obtain the Required Credentials

### 1. Sign in to Strata Cloud Manager
### 2. Create a Custom Role
### 3. Create a Service Account
### 4. Configure the Integration

## Get your Palo Alto Networks Device Security Credentials

> This integration requires that API access be configured.

Reviewer: ```suggestion
This integration requires API access configuration.
```

Author: yes, updated it

> To obtain the **Client ID** and **Client Secret**, please refer to the official **API User Guide**:

Reviewer: ```suggestion
To obtain the **Client ID** and **Client Secret**, see the [API User Guide](https://pan.dev/iot/api/iot-public-api-new).
```

Author: yes, updated it

> https://pan.dev/iot/api/iot-public-api-new

Reviewer: ```suggestion
```

Author: This is required as it provides the official documentation for creating service account and credential generation
